airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -467,20 +454,15 @@ class DataprocCreateClusterOperator(BaseOperator):
     :type impersonation_chain: Union[str, Sequence[str]]
     """
 
-    template_fields = (
-        'project_id',
-        'region',
-        'cluster',
-        'impersonation_chain',
-    )
-
     @apply_defaults
     def __init__(  # pylint: disable=too-many-arguments
         self,
         *,
+        cluster_name: str,
```

Comment:
This backward compatible with 1.10.X:
https://github.com/apache/airflow/blob/827a717fecc675b4d58fa202be003dee8e423632/airflow/contrib/operators/dataproc_operator.py#L192-L193